src/test/java/uk/gov/pay/products/service/ProductCreatorTest.java

```diff
@@ -147,14 +152,60 @@ public void doUpdateByGatewayAccountId_shouldUpdateProduct() {
         verify(mockedProductEntity,  times(1)).setName(updatedName);
         verify(mockedProductEntity,  times(1)).setDescription(updatedDescription);
         verify(mockedProductEntity,  times(1)).setPrice(updatedPrice);
+        verify(mockedProductEntity, times(1)).setReferenceEnabled(true);
```

Comment:
I know you’re just following the conventions of the file but `times(1)` is unnecessary because it’s the default

Reply:
👍